openassessment/xblock/utils/notifications.py

```diff
@@ -34,3 +41,56 @@ def send_staff_notification(course_id, problem_id, ora_name):
         COURSE_NOTIFICATION_REQUESTED.send_event(course_notification_data=notification_data)
     except Exception as e:
         logger.error(f"Error while sending ora staff notification: {e}")
+
+
+def send_grade_assigned_notification(usage_id, ora_user_anonymized_id, score):
+    """
+        Send a user notification for a course for a new grade being assigned
+    """
+    from openassessment.data import map_anonymized_ids_to_usernames as map_to_username
```

Comment:
why `map_anonymized_ids_to_usernames` is imported inside a function ?

Reply:
To avoid circular import errors.